test/fixtures/notify-then-sleep.ts

```diff
@@ -1,3 +1,4 @@
+// @ts-ignore
 export default function (i32array) {
```

Comment:
```suggestion
export default function (i32array: Int32Array) {
```

Reply:
Or wait, is this file used by any test?

Reply:
I have no idea, I just review all imports and found some ts errors.